modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/facet/display/context/builder/AssetCategoriesSearchFacetDisplayContextBuilder.java

```diff
@@ -377,6 +377,10 @@ protected void setTermDisplayContexts(
 	}
 
 	private List<Tuple> _collectBuckets(Facet facet) {
+		if (facet == null) {
```

Comment:
@joshuacords is this check still necessary? seems odd that only category facet would need it?

Reply:
It's probably not necessary any more, but I think the Category and Vocabulary facet were doing permission checking on the facets and that's why just they needed it. (Folders does a followup search to check).
So I don't think it's necessary, but I don't see why we won't want to handle the null situation - do you think I should remove that and test again? @BryanEngler

Reply:
actually it looks like all of the other FacetDisplayContextBuilders are already doing this check, this was the only one that wasnt so should be fine to leave the check there